spec.bs

```diff
@@ -808,6 +826,22 @@ A <dfn export for=fencedframetype>destination event</dfn> is either a
 
      1. Set |destination url| to |destination map|[|eventType|].
 
+    1. If |event|'s [=destination enum event/attributionReportingEnabled=] is true and the result
+       of [=getting supported registrars=] is not [=list/is empty|empty=] and |event|'s
+       [=destination enum event/contextOrigin=] [=check if an origin is suitable|is suitable=]:
+
+        1. If |event|'s [=destination enum event/type=] is `"reserved.top_navigation"`,
+           set |attributionReportingEligibility| to "<code>[=eligibility/navigation-source=]</code>".
+
+        1. Otherwise, set |attributionReportingEligibility| to "<code>[=eligibility/event-source=]</code>".
+
+        1. Set |processResponse| to these steps given a [=response=] |response|:
+
+           1. Run [=process an attribution eligible response=] with |event|'s
```

Comment:
Please make sure that https://wicg.github.io/attribution-reporting-api/#process-an-attribution-eligible-response can handle running in parallel. Right now it does not, because https://wicg.github.io/attribution-reporting-api/#obtain-and-deliver-a-debug-report "queues a task" to no explicit task source, which seems wrong. There may be more wrong with that algorithm, but that's at least my initial reaction, so following-up on that spec might be good.

Reply:
Thanks for catching this, I will file a PR to fix that in attribution reporting spec.